A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -830,6 +848,34 @@ var A320_Neo_UpperECAM;
             this.divMain.appendChild(this.gaugeN1);
             this.allGauges.push(this.gaugeN1);
         }
+        getModeEGTMax(){
+            if(this.throttleMode == ThrottleMode.TOGA && this.timerTOGA > 0){
+                return 1060;
+            }
+            else if(this.throttleMode == ThrottleMode.TOGA && this.timerTOGA <0){
+                return 1025;
+            }
+            if(this.throttleMode == ThrottleMode.FLEX_MCT){
+                return 1025;
+            }
+            if(this.throttleMode == ThrottleMode.CLIMB){
+                return 875;
+            }
+            if(this.ThrottleMode == ThrottleMode.AUTO){
+                return 850;
+            }
+            else{
+                return 750;
```

Comment:
could you add a comment that says when this would trigger? is it for a specific throttle mode?

Reply:
That is for the last remaining mode IDLE/GND Start